File: app/src/main/java/com/gameondigital/gameonapp/Login/ui/LoginActivityContract.java
package com.gameondigital.gameonapp.Login.ui;

public interface LoginActivityContract {
    void setFonts();

    void validateLogin();

    void dismissLoading();

    void showLoginView();

    void callActivity(Class<?> cls);

    void finishActivity();

    void showErrorLogin(String errorCode);
}

File: app/src/main/java/com/gameondigital/gameonapp/Login/ui/LoginActivityContract.kt
package com.gameondigital.gameonapp.Login.ui

import android.widget.EditText

interface LoginActivityContract {
    fun initComponents()
    fun validatedLogin(email: String, password: String)
    fun goToHome()
    fun setValidateLoginError(edtText: EditText, error: String)
    fun showErrorLogin(msgError: String)
    fun showLoginView()
}